### ISSUE TITLE
Front of House Accessibility Information Page

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4694]
We believe readers should have easy access to accessibility information and signposting for accessibility related queries.  

We propose creating a standard static page for journals and presses that provides accessibility information to be included as one of the site options.  This idea grew from a team discussion about #4510 which was about providing accessibility information on downloadable files.

### Functionality
It should:
- be navigable from all Front of House pages.
- include information we have written about the accessibility of the journal theme in use, including downloadable files.  It must be clear whether information is from Janeway, or customised.
- signpost users to report a11y bugs or raise questions.  
- allow journal managers to include their own accessibility information, especially those who use custom styling or have extended the themes.
- at press level, there should be an option which pulls together all the custom journal a11y statements while including the Janeway information just once, (and only those themes in use on the press).

### Example User Stories:

**As a reader who has encountered an accessibility barrier**, I need to be able to determine if it is 'my end' or 'theirs' in order to know what to do about it.  If the issue is with the site, then I would like to be able to report it, but if it is difficult to do so, I may not bother.

**As someone working with students, some of whom have access needs**, I wish to know whether a resource may be accessed by my students.

**As an author writing about disability or disabled people**, I need to know the journal I am submitting to will not publish my article in a format excludes people with that disability from reading the article.

**As journal staff** we wish to maximise our reach and readership.

**As journal staff** our parent organisation has specific requirements for accessibility statements with which we must comply.

**As journal staff** with limited resources, we would like to provide accessibility information but to not have the time, resources or expertise to do so, and likewise as a small team, we are using Janeway without custom modifications.

**As developers on Janeway** we want to know about access barriers so that we can work to reduce them.  In the spirit of `nothing about us without us` we particularly wish to know about barriers from the people who have been affected by them.

**Comment by StephDriver** [comment]
see also [W3C - Tool for Developing an Accessibility Statement](https://www.w3.org/WAI/planning/statements/)

**Comment by StephDriver** [comment]
Include information on how to read articles aloud with text-to-speech, see #2849

**Comment by severett89** [comment]
From my understanding, while it's important to have an accessibility policy set for Janeway in the manner described above, the publisher might like to be able to set a policy here that can be seen across all of its journals. Steph has suggested potentially:

- Janeway Text
- Press/Publisher Text (set at press level)
- Journal Text (only possible if a press manager turns on that setting for that journal)

Which would cater for all tiers of potentially necessary accessibility info according to the platform, publisher and journal.

**Comment by StephDriver** [comment]
Include information about accessibility mode (for all themes? or just for Material/OLH depending on how it is implemented).
